web/src/stream_edit.js

```diff
@@ -668,7 +668,15 @@ export function initialize() {
     });
 
     $("#channels_overlay_container").on("click", ".stream-row", function (e) {
-        if ($(e.target).closest(".check, .subscription_settings").length === 0) {
+        const $target = $(e.target);
+        if (
+            $target.hasClass("subscriber-count") ||
+            $target.hasClass("fa-user-o") ||
```

Comment:
Checking an icon name is fragile; it's pretty likely if the icon gets changed for visual reasons, this code will break by accident.

Can you name the region to target directly via some more semantic CSS class?

Also, do we need any hover behavior or other CSS?